src/data/requestFiltering.js

```diff
@@ -0,0 +1,67 @@
+import { getSelectedSizes } from '../data/sizes'
+import { updateQueryString } from '../data/url'
+
+export function populateFilterFrom(queryStringData) {
+  return {
+    createdBy: { id: queryStringData.creator },
+    provider: { id: queryStringData.provider },
+    size: getSelectedSizes(String(queryStringData.size).toUpperCase()),
+  }
+}
+
+/** NOTE: This should clear all values used by `populateFilterFrom()` */
```

Comment:
is this note still appropriate?

Reply:
Yes, though I might not have worded it well. My goal is to somehow link these two functions so that we remember to change both if we change either.

In other words, if we later add the ability to filter on yet another property, we'll need to update both the `populateFilterFrom()` and the `clearFilter()` functions to set / clear that filter, as appropriate.